lib/money/money/arithmetic.rb

```diff
@@ -1,4 +1,17 @@
 class Money
+  CoercedNumber = Struct.new(:value) do
+    include Comparable
+
+    def +(other) raise TypeError; end
+    def -(other) raise TypeError; end
+    def /(other) raise TypeError; end
```

Comment:
I would say division is valid for coercion as you would not make sense to say divide `$10` by `$2`. You would divide `$10` by `2`.